README.md

```diff
@@ -90,457 +90,467 @@ type: tutorial
 
-![ocp-installedoperators](doc/source/images/ocpinstalledoperators.png)
+    ![ocp-installedoperators](images/ocpinstalledoperators.png)
 
-- Click on `E.D.D.I` operator, under **Provided API's**, click on the first `Create Instance` as shown.
+1. On the E.D.D.I. operator screen, navigate to Provided APIs and select the first **Create Instance**.
 
-![ocp-createdbinstance](doc/source/images/ocpcreateinstance.png)
+    ![ocp-createdbinstance](images/ocpcreateinstance.png)
 
-- The **Create E.D.D.I** page will be displayed with the default YAML, edit the `storageclass_name` in the YAML file, and click on the `Create` button as shown. If the default YAML file is not visiblie you can copy paste the bellow YAML file replacing the `storageclass_name`.
+1. The E.D.D.I page displays with the default YAML. Edit the `storageclass_name` in the YAML file and click on the **Create** button. If the default YAML file is not visiblie, you can copy paste the bellow YAML file replacing the `storageclass_name`.
 
-```yaml
-apiVersion: labs.ai/v1alpha1
-kind: Eddi
-metadata:
-  name: eddi
-spec:
-  size: 1
-  mongodb:
-    environment: prod
-    storageclass_name: <existing_storageclass>
-    storage_size: 20G
-```
+    ```yaml
+    apiVersion: labs.ai/v1alpha1
+    kind: Eddi
+    metadata:
+      name: eddi
+    spec:
+      size: 1
+      mongodb:
+        environment: prod
+        storageclass_name: <existing_storageclass>
+        storage_size: 20G
+    ```
 
-![ocp-createyaml](doc/source/images/ocpcreateyml.png)
+    ![ocp-createyaml](images/ocpcreateyml.png)
 
-- E.D.D.I Operator pods should come up when the installation is completed.
+    E.D.D.I Operator pods should come up when the installation is completed.
 
-- Under the left panel click on `Networking` and select `Routes`, make sure you are in the `eddi-project` namesapace, you can now see the `eddi-route` with a URL as shown.
+1. Under the left panel, select **Networking > Routes**. Make sure you are in the `eddi-project` namesapace. You should now see the `eddi-route` with a URL like the following: 
 
-![ocp-routes](doc/source/images/ocproutes.png)
+    ![ocp-routes](images/ocproutes.png)
 
-- You can now visit the URL to access the E.D.D.I Dashboard.
+1. You can now visit the URL to access the E.D.D.I Dashboard.
 
-![ocp-eddidashboard](doc/source/images/eddidashboard.png)
+    ![ocp-eddidashboard](images/eddidashboard.png)
 
 ### Step 4: Create a chatbot in E.D.D.I
 
-- Now, let's create a chatbot in E.D.D.I from scratch.
+Now, let's create a chatbot in E.D.D.I from scratch.
+
+To create a chatbot, you will complete these steps: <!--EM: I feel like, at the very least, we need to link people to where they can learn more about these concepts (regular dictionary, behavior rules, output sets). Is this common vocab for developing chatbots? Why would readers need to develop these things? This feels so advanced and I'm worried about just dropping our readers into this without more explanatory text -->
```

Comment:
**Is this common vocab for developing chatbots?**
Yes, "dictionary, behavior rules, output sets" are common terminologies. Any person/developer dealing with the data would know it. 

**Why would readers need to develop these things?**
Readers would need to develop these things in-order to setup the chatbot from scratch. Infact these detailed steps will be useful for reader to modify the application according to their need.